.github/workflows/test.yml

```diff
@@ -8,6 +8,7 @@ jobs:
         node:
           - '12'
```

Comment:
No need to test on this anymore if we're upgrading.

Reply:
Removed.